packages/cashc/src/ast/AstBuilder.ts

```diff
@@ -333,6 +337,16 @@ export default class AstBuilder
     return stringLiteral;
   }
 
+  createDateLiteral(ctx: LiteralContext): IntLiteralNode{
+    const rawString = (ctx.DateLiteral() as TerminalNode).text;
+    const stringValue = rawString.substring(6, rawString.length - 2);
+    const timestamp = (Date.parse(stringValue)/1000);
```

Comment:
Do we need to `Math.round()` this to make sure we don't get any decimal values or do we know for sure that this will result in an integer?

Reply:
Yeah, looks like ECMA ISO allows precision up to milliseconds. Added rounding to the nearest second.

Reply:
Perfect. 👍